core/db/migrate/20210714102459_add_deleted_at_to_spree_stores.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToSpreeStores < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spree_stores, :deleted_at, :datetime unless column_exists? :spree_stores, :deleted_at
+    add_index :spree_stores, :deleted_at
```

Comment:
```suggestion
    add_index :spree_stores, :deleted_at unless index_exists?(:spree_stores, :deleted_at)
```

Reply:
We don't need it since old migration won't create this column.
I added `self.paranoia_column = :id` in other migration to fix it.

Reply:
@tomurb we need it because someone could have add this in their projects